kivy/uix/recycleboxlayout.py

```diff
@@ -91,8 +91,6 @@ def _update_sizes(self, changed):
                         xo = posx - (w / 2.)
                 opt['pos'] = [xo, yo]
 
-        return relayout
```

Comment:
Something needs to be returned because it is being used [here](https://github.com/kivy/kivy/pull/4785/files#diff-ce81d7c10bed1f52a454905d0f48131eL101)